### ISSUE TITLE
Include summary of unsupported features

### ISSUE THREAD

**Issue by joepavitt** [FlowFuse/.github#61]
### Description

We should add a "Comment" node into the flow which details any properties/features that are not supported yet in D2.0

We could have a `comments` class/instance passed to each Transformer. Then a function on that class that provides ability to `comments.add` with a requirement to pass in `node` and the specific `comment` 

Aftwr paesing all nodes, we should loop iver the detailed comments, and should then construct a multiline report in the comment node for which properties are not supported, ensuring we clearly label the relevant nodes (by name, ID, label where appropriate)

### Requested By

- Customer name and/or link to HubSpot contact

**Comment by hardillb** [comment]
@joepavitt did you mean to open this here? do you want me to move this to the dbv1 to dbv2 migration app?

**Comment by joepavitt** [comment]
Yeah, sorry this should have gone to migration repo - did it from mobile. I cant transfer from my phone
